Posts/2021/02/Announcing-Community-Blog.md

```diff
@@ -0,0 +1,55 @@
+---
+post_title: Announcing the PowerShell Community Blog
+username: sewhee@microsoft.com
+categories: PowerShell
+tags: Blog news, Announcement
+featured_image:
+summary:
+---
+
+# Announcing the PowerShell Community Blog
```

Comment:
As suggested by Sean Wheeler (You :) ) pull out the H1 title as it echoes the Blog Post Title on viewing

We *could* do something like this (a Summary)
Summary: The PowerShell Community Blog is introduced (although in this case, it might not fit) with the Summary in Bold and the following text normal